dashboard/src/components/AppView/AppView.tsx

```diff
@@ -86,6 +88,26 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     getApp(releaseName, namespace);
   }
 
+  public async componentDidUpdate(prevProps: IAppViewProps) {
+    if (!_.eq(this.props.app, prevProps.app)) {
```

Comment:
Please, avoid using `lodash` for trivial JS operations. This is the same as:

```
if (this.props.app !== prevProps.app) {
```

ECMAScript already includes a lot of new methods to work with Objects and Arrays so most of the lodash helpers are no longer required.